### PR TITLE
Add ProgressBar.style().

### PR DESCRIPTION
**Pull request by andrewchambers**

### CODE REVIEW

File: src/progress_bar.rs [hunk 1]
```diff
@@ -54,6 +54,11 @@ impl ProgressBar {
         }
     }
 
+    /// Get the a clone of current progress bar style.
+    pub fn current_style(self) -> ProgressStyle {
```
Comment (djc) [inline src/progress_bar.rs:58]: Please change the name to `style()`, and return a reference. Then, the caller can clone it.

**Review by djc — COMMENTED** [review]
Hey, thanks for submitting a PR. What's your use case here?

**Comment by andrewchambers** [comment]
The use case is a function that takes a progress bar, wishes to perform an operation, then restore the progress bar to its old style.

File: src/progress_bar.rs (hunk 1 above)
Comment (andrewchambers) [reply src/progress_bar.rs:58]: Is this possible when it is behind a mutex?

**Comment by andrewchambers** [comment]
Sorry I must have been a bit tired, fixed the sloppy comment.

**Review by djc — APPROVED** [review]

File: src/progress_bar.rs (hunk 1 above)
Comment (djc) [reply src/progress_bar.rs]: That's a fair point, it is not!

**Pull request merged**